common/utils.js

```diff
@@ -47,7 +47,9 @@
             title: "Your need to be logged in to continue.",
             message: "To open the login window press"
         },
-        "tableMissing": "No table specified in the form of 'schema-name:table-name' and no Default is set."
+        "tableMissing": "No table specified in the form of 'schema-name:table-name' and no Default is set.",
+        "unauthorizedMessage" : "You are not authorized to Update entities.Please report this problem to your system administrators.",
```

Comment:
Since this is going to be a general message for any 403 errors, I think we should say something in line of "You are not authorized to perform this action." Instead of being too specific and saying "Update entities".

Reply:
Error message has been changed